src/libraries/Directory.Build.props

```diff
@@ -305,8 +309,11 @@
   <PropertyGroup Condition="'$(BuildAllConfigurations)' != 'true'">
     <!-- We add extra binplacing for the test shared framework until we can get hardlinking with the runtime directory working on all platforms -->
     <BinPlaceTestSharedFramework Condition="'$(BuildTargetFramework)' == '$(NetCoreAppCurrent)'">true</BinPlaceTestSharedFramework>
```

Comment:
Do we need to condition this to when we're not targeting mobile? I guess we don't need to binplace the testhost for those scenarios.

Reply:
We originally had it in, but I wasn't sure the scope of changes needed for build-job.yml.

Reply:
Ah right. Do you have the errors you saw? Just curious.

Reply:
No, but I do remember the errors saying it could not copy the testhost folder.  It may be easy, but probably too much for one PR.